app/controllers/confirmations_controller.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+# Create Confirmations for Users
+class ConfirmationsController < Devise::ConfirmationsController
+  respond_to :json
+
+  def new
+    super
+  end
+
+  def create
+    super
+  end
```

Comment:
**suggestion (non-blocking)**: We don't need to define the `new` and `create` actions as we're not customizing them.